docs/docs/end-users/VisualStudio.md

```diff
@@ -12,6 +12,9 @@ The F# Formatting extension sets up Fantomas as the default formatter for F# fil
 
 ### Visual Studio for Mac
 
-Install [fantomas](https://www.nuget.org/packages/fantomas) locally with `dotnet tool install fantomas-tool` and configure it as an [External tool]
-
-![VS Mac external tool window](./docs/vsmac-external-tool.png)
\ No newline at end of file
+1. Install [fantomas](https://www.nuget.org/packages/fantomas) locally with `dotnet tool install fantomas` and configure it as an [External tool]
```

Comment:
Small detail but as long as we are not officially released as `fantomas`, you still need to add `--prerelease` to the install command.
See https://fsprojects.github.io/fantomas/docs/end-users/FAQ.html#Why-the-name-Fantomas